packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -7,16 +7,17 @@ import username from 'username';
 import { setInitialForgeConfig } from '../../util/forge-config';
 import installDepList from '../../util/install-dependencies';
 import { readRawPackageJson } from '../../util/read-package-json';
+import { version } from '../../../package.json';
 
 const d = debug('electron-forge:init:npm');
 
 export const deps = ['electron-squirrel-startup'];
 export const devDeps = [
-  '@electron-forge/cli',
-  '@electron-forge/maker-squirrel',
-  '@electron-forge/maker-zip',
-  '@electron-forge/maker-deb',
-  '@electron-forge/maker-rpm',
+  '@electron-forge/cli@' + version,
```

Comment:
Can we use string templating here 👍 ES6 😄

Reply:
Maybe even a helper method `versionedModule(string)`

Reply:
Just tried to make it performance-first. 😁 I guess it's not the big value in `electron-forge`'s transpile process?

Reply:
Helper method, yeah? Doing it already.

Reply:
Doing it already 👌